tests/test_vembrane.py

```diff
@@ -99,8 +101,18 @@ def test_command(testcase: os.PathLike, backend: Backend):
         with tempfile.NamedTemporaryFile(mode="w+t") as tmp_out:
             args.output = tmp_out.name
             args.backend = backend
-            if args.command == "filter" or args.command == "tag":
-                module = filter if args.command == "filter" else tag
+            if (
+                args.command == "filter"
+                or args.command == "tag"
+                or args.command == "sort"
+            ):
+                match args.command:
+                    case "filter":
+                        module = filter
+                    case "tag":
+                        module = tag
+                    case "sort":
+                        module = sort
```

Comment:
unrelated, but perhaps we can use a dictionary here instead?
```py
commands = {
  "filter": filter,
  …
}
module = commands[args.command]
…
```
and move the handling of the different module groups (filter/tag/sort vs table vs structured/fhir) to their own functions.

Reply:
I agree. But I guess rather in a separate PR?